src/App.js

```diff
@@ -1,38 +1,36 @@
 import React, { Suspense, lazy } from 'react'
 import styled from 'styled-components/macro'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
-import Home from './components/Home/Home'
-import Footer from './components/Footer/Footer'
-import NotFound from './components/NotFound/NotFound'
+import Home from './components/pages/Home'
+import Navigation from './components/Navigation'
+import NotFound from './components/pages/NotFound'
 
-const Details = lazy(() => import('./components/Details/Details'))
-const About = lazy(() => import('./components/About/About'))
+const Details = lazy(() => import('./components/pages/Details'))
+const About = lazy(() => import('./components/pages/About'))
 
 export default function App() {
   return (
     <AppStyled>
       <Router>
-        <Switch>
-          <Route exact path="/">
-            <Home />
-          </Route>
-          <Route exact path="/details">
-            <Suspense fallback={<p>Lade ...</p>}>
-              <Details />
-            </Suspense>
-          </Route>
-          <Route exact path="/about">
-            <Suspense fallback={<p>Lade ...</p>}>
-              <About />
-            </Suspense>
-          </Route>
-          <Route>
-            <Suspense fallback={<p>Lade ...</p>}>
-              <NotFound />
-            </Suspense>
-          </Route>
-        </Switch>
-        <Footer />
+        <main>
+          <Suspense fallback={<p>Lädt...</p>}>
```

Comment:
Looks great. So the other pages are lazy loaded, but Home page gets fetched immediately, even though all pages are wrapped by Suspense?